### ISSUE TITLE
[Stats] - Quartiles graphs

### ISSUE THREAD

**Issue by nrshrivatsan** [envoyproxy/envoy-tools#4]
## TL;DR

- Inline color-coded Quartile [ `p25`,`p50`,`p75`,_`p95`_ ]

## Mock

<img width="1440" alt="envoy-curses-metrics-quartiles" src="https://user-images.githubusercontent.com/5283109/39090660-10512c3a-459a-11e8-8e1f-24086cf86cf4.png">

## Background

- Current visualization is rudimental for operations! 😄  👍 
![image](https://user-images.githubusercontent.com/5283109/39090607-849bcd22-4598-11e8-9563-3ea5dbc53d8c.png)

-  [Quartiles](https://en.wikipedia.org/wiki/Quartile) made navigable via arrow-keys ⬇️ ↔️  would be extremely valuable and insightful for Envoy-Operators.
<img width="1440" alt="envoy-curses-metrics-quartiles" src="https://user-images.githubusercontent.com/5283109/39090660-10512c3a-459a-11e8-8e1f-24086cf86cf4.png">

- Also making these configureable via CLI input for the NODE-JS code would be elegant.

**Comment by mattklein123** [comment]
This would be awesome! Especially now that we have histograms...

**Comment by nrshrivatsan** [comment]
@mattklein123 thanks for taking time to comment.

We have two options:

1. Patch NodeJS code in envoy-curses to render the values as percentiles
1. Patch Envoy-Proxy metrics to also expose Percentiles

Your vote?
